superset.gemspec

```diff
@@ -35,20 +35,20 @@ Gem::Specification.new do |spec|
   ]
 
   # Uncomment to register a new dependency of your gem
-  spec.add_dependency "dotenv", "~> 2.7"
-  spec.add_dependency "json", "~> 2.6"
+  spec.add_dependency "dotenv", ">= 2.0"
+  spec.add_dependency "json", ">= 2.0"
   spec.add_dependency "terminal-table", "~> 4.0"
-  spec.add_dependency "rake", "~> 13.0"
-  spec.add_dependency "rollbar", "~> 3.4"
-  spec.add_dependency "require_all", "~> 3.0"
-  spec.add_dependency "rubyzip", "~> 1.0"
+  spec.add_dependency "rake", ">= 13.0"
+  spec.add_dependency "rollbar", ">= 3.0"
+  spec.add_dependency "require_all", ">= 3.0"
```

Comment:
Does the `require_all` gem really provide much value?

Reply:
I believe it does, yes.
If we remove it, there would be around 70 classes to explicitly require.

```
require_rel "superset" recursively loads all 70 files in lib/superset/ 
(including subdirectories like chart/, dashboard/, dataset/, database/, security/, services/, tag/, sqllab/, enumerations/)
```